src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -459,7 +459,8 @@ private string UFormatDateString(DateTime dateTime)
                             break;
 
                         case 'V':
-                            sb.Append((dateTime.DayOfYear / 7) + 1);
+                            GregorianCalendar calendar = new GregorianCalendar();
```

Comment:
Agree with @iSazonov that in general, we should be using `CurrentCulture`.  Not an expert by any means on this topic, but looking at the Wikipedia page, it seems we should implement the formula they have there?